### ISSUE TITLE
ESLint is ignoring .vue files

### ISSUE THREAD

**Issue by robertmain** [vuejs/eslint-plugin-vue#830]
**Tell us about your environment**	
* **ESLint version:**	v5.14.1
* **eslint-plugin-vue version:**	v5.0.0
* **Node version:**	v10.9.0

**Please show your full configuration:**

```javascript
module.exports = {
    root: true,
    env: {
        browser: true,
        commonjs: true,
        es6: true,
        jest: true,
        node: true,
    },
    'extends': [
        'plugin:vue/essential',
        '@vue/standard',
        '@vue/typescript'
    ],
    rules: {
        'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
        'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
        'semi': ['error', 'always'],
        'quotes': [1, 'single'],
        'comma-dangle': ['error', 'only-multiline'],
        'indent': ['error', 4],
    },
    parserOptions: {
        parser: '@typescript-eslint/parser'
    }
}
```

**What did you do?**	
```vue
<script lang="ts">
import Vue from 'vue'

export default Vue.extend({
  name: 'HelloWorld',
  props: {
    msg: String
  }
})
</script>
```

**What did you expect to happen?**

According to the eslint config above, there (should) be several issues raised with this code
1. Trailing commas missing in `props`
1. Code is using 2 space indentation, not 4
1. Semi-colon missing at the end of the export
1. Semi-colon missing at the end of the `import`

**What actually happened?**
No eslint issues were raised, unless I manually specify `npx eslint . --ext =.vue`, which (in my belief) should not be necessary

**Comment by robertmain** [comment]
Seems related to #327 but, moved here since that issue is closd

**Comment by mysticatea** [comment]
Yes, you have to specify `.vue` extension manually. ESLint doesn't have the functionality to lint files other than `.js` automatically.

**Comment by robertmain** [comment]
... Seriously?

**Comment by mysticatea** [comment]
Let's hope for ESLint 6.0.0.

**Comment by robertmain** [comment]
Seriously?! It seems nuts that you can't change that!
